src/extras/geometries/EdgesGeometry.js

```diff
@@ -86,6 +86,8 @@ THREE.EdgesGeometry = function ( geometry, thresholdAngle ) {
 
 	}
 
+	coords = coords.subarray( 0, index );
```

Comment:
>  I wanted to avoid over-allocating and extra copying

Won't really matter as lots of temporary alloc already already going on there, so it will all get GC'd together; also assume it will be used in setup so wouldn't be adding to the performance critical per frame memory pressure?